engine/flink/components/dev-table/src/main/scala/pl/touk/nussknacker/engine/flink/table/HardcodedSchema.scala

```diff
@@ -0,0 +1,49 @@
+package pl.touk.nussknacker.engine.flink.table
+
+import org.apache.flink.table.api.{DataTypes, Schema}
+import org.apache.flink.table.types.DataType
+import org.apache.flink.types.Row
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypingResult}
+
+object HardcodedSchema {
+
+  import scala.jdk.CollectionConverters._
+
+  val stringColumnName = "someString"
+  val intColumnName    = "someInt"
+
+  val schema: Schema = Schema
+    .newBuilder()
+    .column("someString", DataTypes.STRING())
+    .column("someInt", DataTypes.INT())
+    .build()
+
+  val rowDataType: DataType = DataTypes.ROW(
+    DataTypes.FIELD(stringColumnName, DataTypes.STRING()),
+    DataTypes.FIELD(intColumnName, DataTypes.INT()),
+  )
+
+  val typingResult: TypingResult = Typed.record(Map("someInt" -> Typed[Integer], "someString" -> Typed[String]))
```

Comment:
You can reuse the column names above. Also please move each field to the separate lines